### ISSUE TITLE
Add IBA to captum

### ISSUE THREAD

**Issue by berleon** [pytorch/captum#358]
Hi everyone,

first I want to thank you for this library! It is great to have a place with the most commonly used interpretability methods. We have developed an attribution method that can quantify the importance of areas in bits / pixel. Here is the paper and the reviews: https://openreview.net/forum?id=S1xWh1rYwB  It was accepted for ICLR 2020.  

We developed an easy-to-use library for pytorch (& tensorflow): https://github.com/BioroboticsLab/IBA 

We think it would be a great addition to the many methods already included in captum.
If you are interested in including our method, I could offer to integrate our pytorch code into captum. 

Best,
Leon

cc my co-author: @karl-schulz

**Comment by NarineK** [comment]
Thank you very much for the kind feedback. We definitely would love to support the community and are planning to start with an `awesome list` where people can add their algorithms. Apart from awesome list the next step would be to add an implementation to the contrib package which we will make ready for contributors and as a last step once the algorithms get more popularity we can graduate it and move to the core.

As a starting point we can start from adding it to the awesome list. Then we will review the approach and suggest to add it to contrib package if you agree :)

cc: @caraya10

**Comment by berleon** [comment]
yes, adding it to a `awesome list` would be good first step. Let me know if I can be of any help with the review or with adding it to the contrib package.
